src/config/dynamic-theme-fixes.config

```diff
@@ -2863,6 +2863,41 @@ button
 
 ================================
 
+colors.dopely.top
+
+INVERT
+.footer
+.color-selector-icon
+
+CSS
+input.number-slider {
+    filter: invert(10%) saturate(0%) !important;
+}
+img[src*="svg"], .title {
+    filter: invert(50%) !important;
+}
+.accountDopely__letter,
+.editProfile__changeProfilePicture {
+    filter: saturate(0%) brightness(50%) !important;
+}
+div[class*="text"],
+a[class*="header"],
```

Comment:
```suggestion
div.text,
a.header,
```

Reply:
This is done so that it includes all div with "text" or all a with "header" in their names across the whole site.
Doing it the way you suggested excludes some out. This site is very unorganized with the code and how things are referred as